src/main.rs

```diff
@@ -159,7 +159,7 @@ static ARGS: Lazy<Args> = Lazy::new(|| {
 
 struct Args {
     addrs: Vec<SocketAddr>,
-    #[allow(dead_code)] // only used on unix, so dead code on windows
+    #[cfg_attr(target_os = "windows", allow(dead_code))] // only used on unix, so dead code on windows
```

Comment:
CI says to chage the formatting here. I would also suggest changing to the [`windows` configuration option](https://doc.rust-lang.org/reference/conditional-compilation.html#unix-and-windows) instead.

```suggestion
    // only used on unix, so dead code on windows
    #[cfg_attr(windows, allow(dead_code))]
```